hud-data/thughud.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "ThugHud",
+  "author": "Russ",
+  "contributors": [
+    "N1te (Sex Hud creator)"
+  ],
+  "social": {
+    "support": "https://steamcommunity.com/tradeoffer/new/?partner=413718379&token=3fJl5eiP",
+    "issues": "Add me on discord (russty._)",
```

Comment:
This needs to be a link, can use `https://discordapp.com/channels/@me/USERID`

Reply:
done